products/sle/main.pm

```diff
@@ -712,6 +712,7 @@ elsif (get_var("NFV")) {
 }
 elsif (get_var("REGRESSION")) {
     load_common_x11;
+    load_xen_tests if check_var("REGRESSION", "xen");
```

Comment:
You are only loading these tests on SLE, not openSUSE, right? lib/main_common was intended to be used whenever we run the same tests on both SLE and openSUSE so would you mind to move your test loading functionality from lib/main_common to products/sle/main.pm?